javascript/javascript1/pizza-exercise/pizzaNew.js

```diff
@@ -0,0 +1,16 @@
+const amountOfPizza = prompt("How many pizaas you want to order?");
+const familySizePizza = confirm("Do you like to order family size pizza ? if yes click ok else click cancel : ");
+const pizzaPrice = 60;
+const nameOfPizza = prompt("Enter the name of the pizza ?");
+let totalPrice = amountOfPizza * pizzaPrice;
+if(familySizePizza){
+    doublePrice = pizzaPrice * 2;
+    console.log(`The price of pizaa is double: $${doublePrice}`);
+    console.log(`New pizza order: ${amountOfPizza} pizzas, the name of pizza is : ${nameOfPizza} `);
+    totalNewPrice = doublePrice * amountOfPizza;
+    console.log(`Total cost for the order is : $${totalNewPrice}`);
+}
+else {
+    console.log(`New pizza order: ${amountOfPizza} pizzas, the name of pizza is : ${nameOfPizza} `);
+    console.log(`Total cost for the order is : $${totalPrice}`);
+}
```

Comment:
You could simplify this, since only a few values are changing in the if/else-statement:
```javascript
let totalPrice = amountOfPizza * pizzaPrice;
if (familySizePizza) {
    const doublePizzaPrice = pizzaPrice * 2;
    totalPrice = amountOfPizza * doublePizzaPrice;
    console.log(`The price of pizza is double: $${doublePizzaPrice}`);
}

console.log(`New pizza order: ${amountOfPizza} pizzas, the name of pizza is: ${nameOfPizza}`);
console.log(`Total cost for the order is: $${totalPrice}`);
```

Reply:
yes thats awesome ! i will change it ! thanks